### DISCUSSION TITLE
Supertags (tana) [Apr 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5555] [Discord Threads]
> 🙋‍♂️ **Hioko**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### No. Could you elaborate on your usecase?  
#  
> 🙋‍♂️ **Hioko**  
#### in fact, i havent drafted my use case, but intuitively, i think i am only lacking one feature & that is supertag  
#  
> 🙋‍♂️ **Hioko**  
#### i am sharing link for more details  
#  
> 🙋‍♂️ **Hioko**  
#### https://help.tana.inc/getting-started-and-overview/introduction-to-supertags.html  
#  
> 🙋‍♂️ **Hioko**  
#### https://www.youtube.com/watch?v=JPxYt1RNB7E  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### That looks interesting  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087079270704218243)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
